importer/src/path_canonicalizer.rs

```diff
@@ -79,9 +80,14 @@ impl<'a, T> Folder<T> for Canonicalizer<'a> {
                                 .map(Some)
                                 .transpose(),
                             },
-                            SymbolValue::Expression(mut e) => {
-                                canonicalize_inside_expression(&mut e, &self.path, self.paths);
-                                Some(Ok(SymbolValue::Expression(e)))
+                            SymbolValue::Expression(mut exp) => {
+                                for tne in
+                                    exp.type_name.iter_mut().flat_map(|tn| tn.expressions_mut())
+                                {
+                                    canonicalize_inside_expression(tne, &self.path, self.paths);
+                                }
+                                canonicalize_inside_expression(&mut exp.e, &self.path, self.paths);
```

Comment:
What is this doing?

Reply:
It turns relative name paths to absolute using the import/use statements.